docs/api_index.rst

```diff
@@ -41,6 +41,18 @@ insights.core.context
     :show-inheritance:
     :undoc-members:
 
+insights.core.dr
+----------------
+
+.. automodule:: insights.core.dr
+    :members:
+    :show-inheritance:
+    :undoc-members:
+
+.. autoclass:: insights.core.dr.ComponentType
```

Comment:
@bfahr I added the autoclass for the `:exclude members:` directive because those members showed up twice otherwise.

Reply:
@csams having this entry here causes the `ComponentType` class to show up twice.  It is in the insights.core.dr section and also right below it.  If you check the rendered docs you'll see it.  When you include the insights.core.dr module that includes everything in the module.  I don't think you need to add the autoclass entry.

Reply:
Thanks for the catch. I missed that Sphinx put the class in there twice.

Here's my problem: docstrings on class attributes aren't actually pydoc and so don't show up when you use the pydoc command, even though I think they'd show up in Sphinx. The "Attributes:" section on the class level pydoc shows up in Sphinx and in pydoc, but Sphinx generates two attributes sections - one for the section and another with each attribute without any docs at all.

I was trying to use the :undoc: directive on just the `ComponentType` class to remove the automatic documentation of its attributes since I'm handling them with the Attributes section of the class level pydoc.

Any suggestions on how to fix it?